gulpfile.js

```diff
@@ -84,6 +84,9 @@ gulp.task('test-debug', ['watch-dev'], function(done) {
   new karma.Server({
     configFile: __dirname + '/karma.conf.js',
     browsers: ['PhantomJSCustom', 'Chrome'],
+    preprocessors: {
+      'lib/qt.js': []
+    },
```

Comment:
I just tested it, just `preprocessors: {},` also works. No need to specify the file.
Also, `reporters: ['progress']` would remove the generation of an empty coverage report.